### PR TITLE
Add message status display via new alerts

### PR DESCRIPTION
**Pull request by nikitayutanov**

### CODE REVIEW

File: utils/gear-hooks/src/context/Account.tsx [hunk 1]
```diff
@@ -35,18 +31,24 @@ function AccountProvider({ children }: Props) {
...
 
+  const login = (_account: Account) => {
+    localStorage.setItem(LOCAL_STORAGE.ACCOUNT, _account.address);
+    setAccount(_account);
+  };
+
   const switchAccount = (_account: InjectedAccountWithMeta) => {
     api?.balance
       .findOut(_account.address)
       .then((balance) => getAccount(_account, balance))
```
Comment (Pinkdesu) [inline utils/gear-hooks/src/context/Account.tsx:42]: getAccount сan be taken out of the component

Comment (Pinkdesu) [reply utils/gear-hooks/src/context/Account.tsx:42]: And getBalance too

File: utils/gear-hooks/src/context/Account.tsx [hunk 2]
```diff
@@ -35,18 +31,24 @@ function AccountProvider({ children }: Props) {
...
-      .then(setAccount);
+      .then(login);
   };
 
   const updateBalance = (balance: Balance) => {
```
Comment (Pinkdesu) [inline utils/gear-hooks/src/context/Account.tsx:46]: useCallback

File: utils/gear-hooks/src/context/Account.tsx [hunk 3]
```diff
@@ -35,18 +31,24 @@ function AccountProvider({ children }: Props) {
     decodedAddress: GearKeyring.decodeAddress(_account.address),
   });
 
+  const login = (_account: Account) => {
+    localStorage.setItem(LOCAL_STORAGE.ACCOUNT, _account.address);
+    setAccount(_account);
+  };
+
   const switchAccount = (_account: InjectedAccountWithMeta) => {
```
Comment (Pinkdesu) [inline utils/gear-hooks/src/context/Account.tsx:39]: useCallback

File: utils/gear-hooks/src/context/Account.tsx [hunk 4]
```diff
@@ -35,18 +31,24 @@ function AccountProvider({ children }: Props) {
     decodedAddress: GearKeyring.decodeAddress(_account.address),
   });
 
+  const login = (_account: Account) => {
```
Comment (Pinkdesu) [inline utils/gear-hooks/src/context/Account.tsx:34]: useCallback

File: utils/gear-hooks/src/context/Account.tsx [hunk 5]
```diff
@@ -35,18 +31,24 @@ function AccountProvider({ children }: Props) {
...
       .then((balance) => getAccount(_account, balance))
-      .then(setAccount);
+      .then(login);
   };
 
   const updateBalance = (balance: Balance) => {
     setAccount((prevAccount) => (prevAccount ? { ...prevAccount, balance: getBalance(balance) } : prevAccount));
   };
 
   const logout = () => {
```
[the remark this answers is not in the document]
Comment (nikitayutanov) [reply utils/gear-hooks/src/context/Account.tsx:39]: `useCallback` without anything else could be premature optimization here, cuz those functions are just part of provider's value.

**Pull request merged**